### ISSUE TITLE
why DeserializedError does notimplement StdError?

### ISSUE THREAD

**Issue by oroulet** [roeeshoshani/binary_serde#3]
If trying to use `?`syntax on deserialize I get that error

```

254 |         let new = AxisData::binary_deserialize(&buffer, Endianness::Big)?;
    |                                                                         ^ the trait `StdError` is not implemented for `DeserializeError`
    |
```

Looks like you are using thiserror_no_std crate. Not really sure what to expect here...

**Comment by roeeshoshani** [comment]
I should add a crate feature named `std` which enables the `std` feature of the `thiserror_no_std` crate. Feel free to open a PR which fixes it.

**Comment by roeeshoshani** [comment]
fixed in version `0.1.5`
